### PR TITLE
feat(BeatLeader): add unsupported pages & fixes

### PR DESCRIPTION
**Pull request by joerkig**
## Description 

Added followed and settings page
Fixed small image lightshow not showing
Removed unnecessary code

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Discord_4u54V68RYp
Discord_7gPOrjuK2m
Discord_Hy4MegSvZq

</details>

### CODE REVIEW

File: websites/B/BeatLeader/metadata.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
 		"replay.beatleader.xyz",
 		"royale.beatleader.xyz"
 	],
-	"version": "1.1.1",
+	"version": "1.2.0",
 	"logo": "https://i.imgur.com/TWPjaYR.png",
```
Comment (DarkVillager) [inline websites/B/BeatLeader/metadata.json:17]: While you're at it, lets change to logo to 512x512
```suggestion
	"logo": "https://i.imgur.com/b5HCP1a.png",
```

**Comment by DarkVillager** [comment]
Also here:
https://github.com/PreMiD/Presences/blob/3dcaf1e5d639ddc0564e54dc910ea9f26d426693/websites/B/BeatLeader/presence.ts#L7

File: websites/B/BeatLeader/metadata.json [hunk 2]
```diff
@@ -13,8 +13,8 @@
 		"replay.beatleader.xyz",
 		"royale.beatleader.xyz"
 	],
-	"version": "1.1.1",
-	"logo": "https://i.imgur.com/TWPjaYR.png",
+	"version": "1.2.0",
+	"logo": "https://i.imgur.com/PhIU3mx.png",
```
Comment (Bas950) [inline websites/B/BeatLeader/metadata.json:17]: Please make the logo 512x512

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/B/BeatLeader/metadata.json (hunk 2 above)
Comment (joerkig) [reply websites/B/BeatLeader/metadata.json:17]: > Please make the logo 512x512

The guidelines says "The image is required to have a **minimum** resolution of 512x512 pixels."
I will change it but, if this has changed to a hard requirement of 512x512 would you be so kind as to update the guidelines?

**Review by Bas950 — APPROVED** [review]

**Review by Timeraa — APPROVED** [review]

**Pull request merged**